src/ompl/util/src/RandomNumbers.cpp

```diff
@@ -44,6 +47,24 @@
 #include <boost/scoped_ptr.hpp>
 #include <boost/date_time/posix_time/posix_time.hpp>
 #include <boost/math/constants/constants.hpp>
+#include <boost/random/uniform_on_sphere.hpp>
+// For boost::make_shared
+#include <boost/make_shared.hpp>
+// For pre C++ 11 gamma function
+#include <boost/math/special_functions/gamma.hpp>
+// For boost::numeric::ublas::shallow_array_adaptor:
+#include <boost/numeric/ublas/vector.hpp>
+
+// The Eigen Includes:
+// The core
+#include <Eigen/Core>
```

Comment:
So this is a new dependency. Currently, OMPL depends only on boost. We've been thinking of adding a matrix library as a dep. I am OK with eigen, but I think Mark has looked at other options as well and may have a different suggestion. That said, if we could avoid the use of Eigen without much difficulty, I would vote for that. If not, maybe it is time to accept it as a dep?